cohort_4/playbook/ACTIVITIES.md

```diff
@@ -756,3 +760,78 @@ the technical mentors for Hack Camp will be pinged. For example:
 If you'd like to be added/removed from the list, let
 [@zachlatta](https://github.com/zachlatta) know and he'll get right on it (say
 `slackbot help` somewhere in the chat to see who's currently on the list).
+
+# Communicating with the Group
+
+In this modern day and age, we're fortunate to have _not one_, _**not two**_,
+**but THREE** methods of communication in camp.
+
+1. Announcing to the whole group
+2. Sharing an announcement to each hacker individually
+3. Using `@channel` on Slack
+
+_But when do we use these marvelous methods of communication, you ask?_
+
+## Announcing to the Whole Group
```

Comment:
Maybe include examples of announcements to the whole group, like on lines 832-837.